covidX/settings/dev.py

```diff
@@ -0,0 +1,4 @@
+from .base import *
+
+DEBUG = True
+CONFIG_FILE = "apps/auth_zero/config/test_config.ini"
```

Comment:
Should I move this config file out of the auth_zero directory and into the common directory?

Reply:
@milan-tom  no it should be part of configurations; may be the yaml config file in config/ and then for yaml file loader, it only loads specific to DEV/PREPROD/PROD settings